Sources/Rendering/Misc/FullScreenRenderWindow/index.js

```diff
@@ -94,8 +94,14 @@ function vtkFullScreenRenderWindow(publicAPI, model) {
   model.interactor.initialize();
   model.interactor.bindEvents(model.container);
 
-  // Expose background
-  publicAPI.setBackground = model.renderer.setBackground;
+  publicAPI.setBackground = macro.chain(
+    publicAPI.setBackground,
```

Comment:
The setter previously existing was supposing that one can give an array of size 3 instead of size 4 (which is the case for the setBackground of renderer) with a default value.

Reply:
if you want to support array of 3 and 4, then rely on the `setBackground()` of the `renderer`.